_pages/index.md

```diff
@@ -3,16 +3,38 @@
+Even two researchers working in the same lab can opt to arrange their data in a different way.
+Lack of consensus (or a standard) leads to misunderstandings and time wasted on rearranging data or rewriting scripts expecting certain structure.
+With the Brain  Imaging Data Structure (BIDS), we describe a simple and easy to adopt way of organizing neuroimaging and behavioral data.
 
-![](./assets/img/dicom-reorganization-transparent-white_1000x477.png){: .center-image }
+![BIDS-folder-organization](./assets/img/dicom-reorganization-transparent-white_1000x477.png){: .center-image }
 
-BIDS is heavily inspired by the format used internally by the OpenfMRI repository (now known as [OpenNeuro.org](http://openneuro.org)). While working on BIDS we consulted many neuroscientists to make sure it covers most common experiments, but at the same time is intuitive and easy to adopt. The specification is intentionally based on simple file formats and folder structures to reflect current lab practices and make it accessible to a wide range of scientists coming from different backgrounds.
+BIDS is heavily inspired by the format used internally by the OpenfMRI repository (now known as [OpenNeuro.org](http://openneuro.org)).
+While working on BIDS we consulted many neuroscientists to make sure it covers most common experiments, but at the same time is intuitive and easy to adopt.
+The specification is intentionally based on simple file formats and folder structures to reflect current lab practices and make it accessible to a wide range of scientists coming from different backgrounds.
 
-Find a good introduction to the BIDS standard in the [paper published in Nature Scientific Data](http://www.nature.com/articles/sdata201644).
+## BIDS is a community effort
 
-Look through some of the community's [presentations on BIDS](https://osf.io/yn93h/).
+BIDS is developed by the community for the community and everybody can [become a part of the community](https://bids.neuroimaging.io/get_involved.html).
 
-Take a look at how the community [uses BIDS](https://medium.com/stanford-center-for-reproducible-neuroscience/bids-usage-survey-results-72637ff039c4).
+## Specification vs. Ecosystem
 
-Sign up to receive occasional [updates by email](https://docs.google.com/forms/d/1ZLi5qRTuX11KGK7qIidSdZvznFoXAqr2wh6003okv-0/edit) and follow BIDS on [Twitter](https://twitter.com/BIDSstandard?ref_src=twsrc%5Etfw).
+Since the inception of BIDS as a specification on how to organize neuroimaging data, a large ecosystem of tools and ressources has evolved around BIDS.
+
+A few of the key elements of this ecosystem are the [BIDS-validator](https://github.com/bids-standard/bids-validator), to automatically check datasets for adherence to the specification, [OpenNeuro](https://openneuro.org/), as a database for BIDS formatted datasets, and [BIDS-Apps](https://doi.org/10.1371/journal.pcbi.1005209), a collection of portable neuroimaging pipelines that understand BIDS datasets.
```

Comment:
perhaps adding BIDS-examples? This could be in the place of OpenNeuro to keep the bids branding consistent

Reply:
I could add BIDS examples on top. I'd rather keep OpenNeuro because it shows that the ecosystem goes beyond things that start with BIDS

Reply:
I agree with illustrating how the ecosystem extends beyond BIDS. Though, I thought about it as the BIDS ecosystem rather than other platforms that use BIDS (or extending out to [BIDS usage in centers](https://docs.google.com/spreadsheets/d/1aySjPpEGGQwFcOavkQdcvk2t2UMXt_zoTzWLWUmq20M/edit#gid=0)) in an effort to highlight our specific BIDS resources. This may conflate BIDS resources with platforms that use BIDS and potentially be confusing to newer users

Reply:
Okay, sounds fine with me :) let's hear one more opinion ... if nobody chimes in, I'll change it and commit with your proposal.

Reply:
I consider openneuro more valuable here than bids-examples. The the first is more relevant for neuroscientists, the latter are more valuable for developers (which - by the time that they start contributing - probably know about it)